tests/requirements.txt

```diff
@@ -0,0 +1,2 @@
+pytest
```

Comment:
we should use the existing pyproject.toml for those dependencies, you can use `poetry add pytest pytest-asyncio` to make the changes there or I can do it after the merge.

Reply:
This is my first time using poetry, that command you provided generate `poetry.lock` file, should I commit it as well?